server/src/main/java/io/deephaven/server/table/stats/ObjectChunkedStats.java

```diff
@@ -40,76 +38,67 @@ public Table processChunks(final RowSet rowSet, final ColumnSource<?> columnSour
         long count = 0;
         int uniqueCount = 0;
 
-        final TObjectLongHashMap<Object> countValues = new TObjectLongHashMap<>();
+        final TObjectLongHashMap<Object> countValues = new TObjectLongHashMap<Object>();
```

Comment:
You could make this cleaner without unduly stressing the replication if you used the interface type on the LHS.
```suggestion
        final TObjectLongMap<Object> countValues = new TObjectLongHashMap<>();
```